cpp/log/etcd_consistent_store-inl.h

```diff
@@ -789,6 +879,79 @@ util::StatusOr<int64_t> EtcdConsistentStore<Logged>::CleanupOldEntries() {
 }
 
 
+template <class Logged>
+void EtcdConsistentStore<Logged>::StartEtcdStatsFetch() {
+  if (etcd_stats_task_.task()->CancelRequested()) {
+    etcd_stats_task_.task()->Return(
+        util::Status(util::error::CANCELLED, "Cancelled."));
```

Comment:
A nicer message is kind of nice, but I'd stick to just ```util::Status::CANCELLED```, this is a pattern that happens a lot, best to keep it low-profile.